app/styles/theme.tsx

```diff
@@ -122,6 +121,17 @@ const theme = createTheme({
   shape: {
     borderRadius: 5,
   },
-});
+};
 
-export default theme;
+// Temporarily comment out the export to prevent TS2345 error
+export default createTheme({
+  ...themeOptions,
+  components: {
+    ...themeOptions.components,
+    MuiSkeleton: {
+      defaultProps: {
+        animation: "pulse", // Change to "pulse" or remove this line to use default
+      },
+    },
+  },
+});
```

Comment:
```suggestion
export default createTheme(themeOptions);
```
I've already changed `MuiSkeleton`'s default animation to `"wave"` in [here](https://github.com/jake-anto/kwiktionary/blob/391fdb935c8b8efd802b00bc0a3cac35286b8329/app/styles/theme.tsx#L47).